### ISSUE TITLE
empty multiline-html-element-content-newline & singleline-html-element-content-newline

### ISSUE THREAD

**Issue by armano2** [vuejs/eslint-plugin-vue#682]
### Tell us about your environment
* **ESLint version:**	5.9.0
* **eslint-plugin-vue version:**	5.0.0-beta.5
* **Node version:**	8.12.0

### Please show your full configuration:
```json
{
  "extends": {
    "plugin:vue/strongly-recommended"
  }
}
```

### What did you do?
**multiline-html-element-content-newline**
```vue
<div
  class="a"
  v-if="b"
  @click.stop.prevent
></div>
```
**singleline-html-element-content-newline**
```vue
<slot name="header"></slot>
```

### What did you expect to happen?
When component has no content this rule should not enforce putting new line.

### What actually happened?
```
 Expected 1 line break after opening tag (`<div>`), but no line breaks found
 Expected 1 line break after opening tag (`<div>`), but no line breaks found
```

**Comment by armano2** [comment]
I'm unsure if this should be option or defaul behavior, but its common to use `<slot></slot>` without content ;>

**Comment by michalsnik** [comment]
I agree, this makes sense and it shouldn't report if there is no content inside.
